### ISSUE TITLE
Missing url for `Learn more about blocking a user.`

### ISSUE THREAD

**Issue by yp05327** [go-gitea/gitea#29992]
### Description

@KN4CK3R 
I think we forgot a link here?
![image](https://github.com/go-gitea/gitea/assets/18380374/1f0fff7c-adad-457d-97d4-2cd3aa63f2ab)
related #29028

### Gitea Version

latest

### Can you reproduce the bug on the Gitea demo site?

Yes

### How are you running Gitea?

build

### Database

None

**Comment by KN4CK3R** [comment]
No, it's just a reminder to learn more! Yes, missing link, will send a PR.

**Comment by wxiaoguang** [comment]
> No, it's just a reminder to learn more! Yes, missing link, will send a PR.

Will be fixed by Add URL to Learn more about blocking a user. #37355
